components/scream/src/share/field/field_tag.hpp

```diff
@@ -37,7 +37,7 @@ enum class FieldTag {
   Variable
 };
 
-inline std::string tag2string (const FieldTag ft) {
+inline std::string e2str (const FieldTag ft) {
```

Comment:
I believe that the scorpio interface code uses tag2string.  Just noting that here so that when that PR goes in we will likely need to update the scorpio code in this PR or the code won't build.

Reply:
or alternatively, if this goes in first then update the SCORPIO code...

Reply:
I realized that for all other enums I use the name `e2str`, so I figured I uniformed the style and use the same fcn name here. It's easier to remember in a random piece of code: you have an enum `e`? then probably `e2str(e)` will give you what you want.